bedrock/firefox/templates/firefox/includes/hub/value-cards.html

```diff
@@ -0,0 +1,69 @@
+{#
+ # I'm putting this in the includes folder for reference, though it
+ # probably wouldn't really work as an include and you might just
```

Comment:
Yeah, it's not used, it was just a carry-over from the hub-common branch. If none of us are actually using it as an include (which we probably shouldn't be) we can delete it.